Gemfile

```diff
@@ -36,7 +36,7 @@ gem 'jwt', '~> 2.2'
 gem 'jwt-multisig', '~> 1.0'
 gem 'bunny'
 gem 'phonelib',     '~> 0.6.0'
-gem 'twilio-ruby',  '~> 5.6.0'
+gem 'twilio-ruby',  '~> 5.25.4'
```

Comment:
Why decreasing the Twilio version?
We should find a way to work with new versions.
We can't be stuck at this version forever

Reply:
Uh, its 25 vs 6
Sorry)